plugins/BEdita/Core/src/ORM/QueryFilterTrait.php

```diff
@@ -126,4 +97,63 @@ public function fieldsFilter(Query $query, array $options)
             return $exp;
         });
     }
+
+    /**
+     * Get query expression for an operator on a field with a value.
+     * Unrecognized operators are ignored and have no effect.
+     *
+     * @param QueryExpression $exp Current query expression
+     * @param string $operator Filter operator
+     * @param string $field Filter field
+     * @param string $value Filter value
+     * @return QueryExpression Operator query expression
+     */
+    protected function operatorExpression(QueryExpression $exp, $operator, $field, $value)
+    {
+        switch ($operator) {
+            case 'eq':
+            case '=':
+                $exp = $exp->eq($field, $value);
+                break;
+
+            case 'neq':
+            case 'ne':
+            case '!=':
+            case '<>':
+                $exp = $exp->notEq($field, $value);
+                break;
+
+            case 'lt':
+            case '<':
+                $exp = $exp->lt($field, $value);
+                break;
+
+            case 'lte':
+            case 'le':
+            case '<=':
+                $exp = $exp->lte($field, $value);
+                break;
+
+            case 'gt':
+            case '>':
+                $exp = $exp->gt($field, $value);
+                break;
+
+            case 'gte':
+            case 'ge':
+            case '>=':
+                $exp = $exp->gte($field, $value);
+                break;
+
+            case 'null':
+                if (empty($value) || $value === '0') {
+                    $exp = $exp->isNotNull($field);
+                } else {
```

Comment:
In this case `filter[field][null]=gustavo` results in `field IS NULL`.
Is it correct or we should only accept `'1'`, `'true'` and `true`?

Reply:
Right. I've used `filter_var($value, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE);` now. Only `1`, `'1'`, `'true'`, `true`, `0`, `'0'`,`false` and `'false'` are accepted.
Otherwise filter is ignored.